src/components/molecules/ProfilePictureInput/ProfilePictureInput.tsx

```diff
@@ -42,6 +44,16 @@ const ProfilePictureInput: React.FunctionComponent<PropsType> = ({
   const [error, setError] = useState("");
   const firebase = useFirebase();
   const uploadRef = useRef<HTMLInputElement>(null);
+  const queryParams = useQuery();
+  const venueId = queryParams.get("venueId");
+
+  const customAvatars = useAsync(async () => {
+    const storageRef = firebase.storage().ref();
+    const list = await storageRef.child(`/assets/avatars/${venueId}`).listAll();
+    const promises = list.items.map((item) => item.getDownloadURL());
+    const avatars: string[] = await Promise.all(promises);
```

Comment:
May as well just inline these:

```suggestion
    const avatars: string[] = await Promise.all(list.items.map((item) => item.getDownloadURL()));
```

Reply:
Changed in: https://github.com/sparkletown/sparkle/commit/6cdbdb2d81f61c67a0665f91bf358d09fc89bfbe